conans/model/editable_cpp_info.py

```diff
@@ -1,11 +1,14 @@
 # coding=utf-8
 import os
 from collections import OrderedDict
+from io import StringIO
```

Comment:
Probably better to use `six.StringIO`. Change also the import from `utils.tools.py` (not related to this PR, but please do it)